service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -337,6 +354,12 @@ private boolean onlineSchemaIsModifiable() {
         || ServiceRegistryConfig.INSTANCE.isAlwaysOverrideSchema();
   }
 
+  private boolean isIllegalValue(String context) {
```

Comment:
delete this method because should be ` is valid value`

Reply:
done